### DISCUSSION TITLE
Q3 of Day13

### DISCUSSION THREAD

**Discussion by Mang-Joo** [I-JUNNYLAND-I/algorithm#82] [Q&A] [answered]
접근 방법을 어떻게 해야 할 지 모르겠씁니다.

**Comment by wlsdn93** [comment, marked as answer]
문자열이 Nice 한지 판단하는 로직과 substring을 만드는 로직을 나눠야 합니다. 
substring을 만드는건 단순 2중 포문으로 접근해보세요.
